File: 3600-3699/3689. Maximum Total Value of Array After K Operations/Solution.java
class Solution {
    public long maxTotalValue(int[] nums, int k) {
        int lo = nums[0], hi = nums[0];
        for (int x : nums) {
            if (x < lo) lo = x;
            if (x > hi) hi = x;
        }
        return (long)(hi - lo) * k;
    }
}

File: 3600-3699/3689. Maximum Total Value of Array After K Operations/Solution.cpp
class Solution
{
public:
  long long maxTotalValue(vector<int> &nums, int k)
  {

    int minimumValue = nums[0];
    int maximumValue = nums[0];

    for (int value : nums)
    {

      minimumValue = min(minimumValue, value);
      maximumValue = max(maximumValue, value);
    }

    return 1LL * (maximumValue - minimumValue) * k;
  }
};